src/main/java/org/broadinstitute/hellbender/engine/FeatureContext.java

```diff
@@ -290,4 +289,33 @@ private int windowStop(final SimpleInterval locus, final int windowTrailingBases
     private <T extends Feature> List<T> subsetToStartPosition(final Collection<T> features, final int start) {
         return features.stream().filter(feat -> feat.getStart() == start).collect(Collectors.toList());
     }
+
+    /**
+     * Since funcotation factories need an instance of {@link FeatureContext} to funcotate, this convenience method can
```

Comment:
Don't mention Funcotator in the docs for this engine method.

Reply:
Fixed.